### PR TITLE
Provide gnd beacon

### LINKED ISSUE #685: Update hbz BEACON file
At https://de.wikipedia.org/wiki/Wikipedia:BEACON#Bibliotheken,_Verb%C3%BCnde,_Archive there is a BEACON file built from a very outdated open data dump from 2012 that only contained a small part of the union catalogue. As those BEACOn files are used in different service by Wikimedia as well as by others (e.g. https://www.deutsche-biographie.de) the effort to make such an offer is worth it.

We should  think about offering frequently updated versions of a hbz BEACON for

1. resources _by_ a person identified with a specific ID (as the current one) and
2. resource _about_ a person identified with a specific GND ID.

As target both files  should use links to lobid and not the Aleph OPAC.

### PR DESCRIPTION
**Pull request by dr0i**
Get the data from lobid. Create a beacon.

Resolves #685.

### CODE REVIEW

**Comment by dr0i** [comment]
Crontabed it to produce a beacon every 13th of a month. Beacon will be copied to `https://lobid.org/download/beacons/hbzlod-pndbeacon.txt` . Don't know what the `#EXAMPLES` in the beacon head means, just copied it from the original findbuch-file.

File: src/main/resources/beacon/data/beacon_head.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#FORMAT: BEACON
+#VERSION: 0.1
+#NAME: GND-Personen im Verbundkatalog des HBZ (Linked Open Data Bereitstellungen)
+#MESSAGE: HBZ-Verbundkatalog
+#PREFIX: http://d-nb.info/gnd/
+#TARGET: http://http://lobid.org/resources/search?agent=http%3A%2F%2Fd-nb.info%2Fgnd%2F{ID}
```
Comment (acka47) [inline src/main/resources/beacon/data/beacon_head.txt]: Please fix the URL (double "http"). Also, why not use the https URL?

File: src/main/resources/beacon/data/beacon_head.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+#DESCRIPTION: Personen der Formalerschliessung (Tn und Tp) mit GND-Nummer "extern" erstellt aus Linked Open Data
```
Comment (acka47) [inline src/main/resources/beacon/data/beacon_head.txt]: This should read:

> Personen und Körperschaften der Formalerschließung monatlich erstellt mit https://lobid.org/resources

File: src/main/resources/beacon/data/beacon_head.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#FORMAT: BEACON
+#VERSION: 0.1
+#NAME: GND-Personen im Verbundkatalog des HBZ (Linked Open Data Bereitstellungen)
```
Comment (acka47) [inline src/main/resources/beacon/data/beacon_head.txt]: > GND-Personen und -Körperschaften im hbz-Verbundkatalog (lobid-resources)

**Comment by dr0i** [comment]
@acka47 fixed the beacon head

**Comment by dr0i** [comment]
Deployed tp production, closing.

**Comment by dr0i** [comment]
Checked if data is indeed created - it is.
Noticed wrong (not-updated)  values for `REVISIT` and `TIMESTAMP`.

**Comment by dr0i** [comment]
Fixed updating of TIMESTAMP and REVISIT with https://github.com/hbz/lobid-resources/commit/1fafc031e3ae16cb803d9a89089a3676025c6a9a.

**Pull request closed without merging**